src/views/Playgrounds/PlaygroundForm.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { useState } from 'react'
+import { useForm, Controller } from 'react-hook-form'
+import {
+  Button as _Button,
+  Text,
+  FormField,
+  TextField,
+  Select,
+  Checkbox,
+  Textarea,
+  HeaderTextField,
+} from '@/shared/components'
+import { textFieldValidation, requiredValidation } from './formValidationOptions'
+import { SelectedItem } from '@/shared/components/Select'
+import styled from '@emotion/styled'
+
+const items: SelectedItem[] = [
+  { name: 'Public (Anyone can see this video', value: 'public' },
+  { name: 'Private', value: 'private' },
+]
+
+type Inputs = {
+  title: string
+  videoSelect: string
+  header: string
+  check: string
+  textarea: string
+}
+
+const PlaygroundForm = () => {
+  const { register, handleSubmit, control, setValue, errors } = useForm<Inputs>({
```

Comment:
Im not super experienced with `react-hook-form` but I think we could omit the `Inputs` type and set `defaultValue` like this: 
```js
    defaultValues: {
      title: '',
      videoSelect: '',
      header: '',
      textarea: '',
    },
```
IMO this will be little bit readable and intuitive